config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
```

Comment:
What is Cable?

Reply:
apparently the config/cable.yml is used by action cable of the rails framework, which isn't relevant since an API app doesn't utilize websocket for real time feature such as chat